normalize/coll.py

```diff
@@ -255,10 +257,10 @@ def itervalues(self):
         return (v for k, v in self.itertuples())
 
     def keys(self):
-        return self._values.keys()
+        return list(self._values.keys())
```

Comment:
I don't think we should convert this to a list.
If `DictCollection` is implementing an interface that's similar to `dict` I'd expect it to behave like a Python 2 `dict` in Python 2 and a Python 3 `dict` in Python 3